.circleci/config.yml

```
@@ -91,6 +91,16 @@ jobs:
           paths:
             - html
 
+      - run:
+          name: make result
+          command: |
+            benchopt run benchopt/tests/test_benchmarks/dummy_benchmark -d Simulated[n_features=100,n_samples=100,rho=0]  -o "*[reg=0.1]" -s python-pgd -r 1 -n 10 --output artifact
```

Maybe a bit better to have a curve with area.
```suggestion
            benchopt run benchopt/tests/test_benchmarks/dummy_benchmark -d Simulated[n_features=100,n_samples=100,rho=0]  -o "*[reg=0.1]" -s python-pgd -r 5 -n 15 --output artifact
```

Also we might want to add 2 more solvers to see a legend a bit fuller?